src/elm/Nav.elm

```diff
@@ -0,0 +1,156 @@
+-}
+navButton : PartialModel a -> NavActions msg -> Html msg
+navButton model actions =
+    case model.page of
+        Pages.Overview ->
+            a
+                [ class "button"
+                , class "-outline"
+                , Util.testAttribute "repo-enable"
+                , Routes.href <| Routes.AddRepositories
+                ]
+                [ text "Add Repositories" ]
+
+        Pages.AddRepositories ->
+            button
+                [ classList
+                    [ ( "button", True )
+                    , ( "-outline", True )
+                    ]
+                , onClick actions.fetchSourceRepos
+                , disabled (model.sourceRepos == Loading)
+                , Util.testAttribute "refresh-source-repos"
+                ]
+                [ case model.sourceRepos of
+                    Loading ->
+                        text "Loading…"
+
+                    _ ->
+                        text "Refresh List"
+                ]
+
+        Pages.RepositoryBuilds org repo maybePage maybePerPage ->
+            div [ class "buttons" ]
+                [ starToggle org repo actions.toggleFavorite <| isFavorited model.user <| org ++ "/" ++ repo
+                , a
+                    [ class "button"
+                    , class "-outline"
+                    , Util.testAttribute <| "goto-repo-hooks-" ++ org ++ "/" ++ repo
+                    , Routes.href <| Routes.Hooks org repo maybePage maybePerPage
+                    ]
+                    [ text "Hooks" ]
+                , a
+                    [ class "button"
+                    , class "-outline"
+                    , Util.testAttribute <| "goto-repo-settings-" ++ org ++ "/" ++ repo
+                    , Routes.href <| Routes.Settings org repo
+                    ]
+                    [ text "Repo Settings" ]
+                ]
+
+        Pages.Settings org repo ->
+            div [ class "buttons" ]
+                [ starToggle org repo actions.toggleFavorite <| isFavorited model.user <| org ++ "/" ++ repo
+                , button
+                    [ classList
+                        [ ( "button", True )
+                        , ( "-outline", True )
+                        ]
+                    , onClick <| actions.refreshSettings org repo
+                    , Util.testAttribute "refresh-repo-settings"
+                    ]
+                    [ text "Refresh Settings"
+                    ]
+                ]
+
+        Pages.Build org repo buildNumber _ ->
+            button
+                [ classList
+                    [ ( "button", True )
+                    , ( "-outline", True )
+                    ]
+                , onClick <| actions.restartBuild org repo buildNumber
+                , Util.testAttribute "restart-build"
+                ]
+                [ text "Restart Build"
+                ]
+
+        Pages.Hooks org repo _ _ ->
+            div [ class "buttons" ]
```

Comment:
this code was updated as well as moved, to fix some missed css cleanup from "-inverted" to "-outline"